phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -330,16 +336,20 @@ protected QueryPlan optimizeQuery(CompilableStatement stmt) throws SQLException
     
     protected PhoenixResultSet executeQuery(final CompilableStatement stmt, final QueryLogger queryLogger)
             throws SQLException {
-        return executeQuery(stmt, true, queryLogger, false);
+        return executeQuery(stmt, true, queryLogger, false, this.validateLastDdlTimestamp);
```

Comment:
validateLastDDlTimestamp is a member field. Why pass it as a function argument ?

Reply:
yeah, used to implement one retry